File: POOUber/Python/carAdvanced.py
from car import Car

class CarAdvanced(Car):
    typeCarAccepted = []
    seatsMaterial   = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super(CarAdvanced, self).__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial   = seatsMaterial

File: POOUber/JS/CarAdvanced.js
class CarAdvanced extends Car {

    constructor(license, driver, typeCarAccepted, seatsMaterial) {

        super(license, driver)
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;

    }

    printCarInfo() {

        console.log(`License → ${this.license}`);
        console.log(`Driver Name → ${this.driver.name}`);
        console.log(`Driver Document → ${this.driver.document}`);

        for (const type of typeCarAccepted) {

            console.log(`Type → ${type}`);

        }

        console.log("Seats Material ↓");
        for (const material of this.seatsMaterial) {
            
            console.log(material);
        }

    }
    
}